tar.go

```diff
@@ -29,11 +30,13 @@ func NewTarStreamer(r io.Reader, keywords []string) Streamer {
 		tarReader:  tar.NewReader(pR),
 		keywords:   keywords,
 	}
-	go ts.readHeaders() // I don't like this
+
+	go ts.readHeaders()
```

Comment:
@cyphar as for `time` vs. `tar_time`, it is more to remove the guessing of where the value of time came from, because that context affects the accuracy of comparison.
When is it that you would have both? I don't think that ought to happen.

If the manifest is built from a tar archive, then the `tar_time` is only precise to the seconds. So when validating an on-disk path whose mtime will be likely to the nanosecond precision, then the comparison ought to be fuzzy.
And the inverse is true.